### PR TITLE
Iot class clarification

### PR DESCRIPTION
**Pull request by MattiasC**
I have tried to clarify the documentation a bit, especially for newbies like me who is trying to make a custom integration conform to HACS standards for the first time.

Let me know if anything is unclear or if something needs to be modified further.

### CODE REVIEW

File: documentation/publish/integration.md [hunk 1]
```diff
@@ -36,41 +33,40 @@ README.md
...
 - `codeowners`
 - `name`
 
-### GitHub releases (optional)
+Check the official Home Assistant [documentation](https://developers.home-assistant.io/docs/en/creating_integration_manifest.html) for the values of those keys.
 
-#### If there are releases
 
-When installing/upgrading it will scan the content in the latest release.
+### Python prerequisites
```
Comment (ludeeus) [inline documentation/publish/integration.md:55]:
```suggestion
### Python prerequisites

```

File: documentation/publish/integration.md [hunk 2]
```diff
@@ -36,41 +33,40 @@ README.md
...
 
-If there are multiple releases in the repository the user have some options to install a specific version.
-The choices will be the last 5 releases and the default branch.
 
-#### If there are no releases
+### Home Assistant Brands
+You must have added your integration to [home-assistant/brands](https://github.com/home-assistant/brands) conform to the UI standards in Home Assistant.
+
+
+### GitHub releases (optional)
```
Comment (ludeeus) [inline documentation/publish/integration.md:63]:
```suggestion
### GitHub releases (optional)

```

File: documentation/publish/integration.md [hunk 3]
```diff
@@ -36,41 +33,40 @@ README.md
...
 
-When installing/upgrading it will scan the content in the latest release.
+### Python prerequisites
+If your integration requires other python libraries, your integration must be added to [home-assistant/wheels-custom-integrations](https://github.com/home-assistant/wheels-custom-integrations).
 
-If there are multiple releases in the repository the user have some options to install a specific version.
-The choices will be the last 5 releases and the default branch.
 
-#### If there are no releases
+### Home Assistant Brands
```
Comment (ludeeus) [inline documentation/publish/integration.md:59]:
```suggestion
### Home Assistant Brands

```

File: documentation/publish/integration.md [hunk 4]
```diff
@@ -4,20 +4,18 @@ title: Integrations
...
 ### Repository structure
 
-- There is only one integration (one directory under `ROOT_OF_THE_REPO/custom_components/`) per repository (if you have more, only the first one will be managed).
-- The integration (all the Python files for it) are located under `ROOT_OF_THE_REPO/custom_components/INTEGRATION_NAME/`.
+- There must only be one integration per repository, i.e. there can only be one subdirectory to `ROOT_OF_THE_REPO/custom_components/`. *If there are more than one, only the first one will be managed.*
+- All files required for the integration to run must be located inside the directory `ROOT_OF_THE_REPO/custom_components/INTEGRATION_NAME/`.
 
-#### OK example:
 
-```text
```
[the remark this answers is not in the document]
Comment (MattiasC) [reply documentation/publish/integration.md:20]: Change reverted and commited.

File: documentation/publish/integration.md [hunk 5]
```diff
@@ -26,8 +24,7 @@ README.md
 ```
 
 #### Not OK example (1):
-
-```text
+```
```
[the remark this answers is not in the document]
Comment (MattiasC) [reply documentation/publish/integration.md:29]: Change reverted and commited.

File: documentation/publish/integration.md [hunk 6]
```diff
@@ -36,41 +33,40 @@ README.md
 ```
 
 #### Not OK example (2):
-
-```text
```
[the remark this answers is not in the document]
Comment (MattiasC) [reply documentation/publish/integration.md:40]: Change reverted and commited.

File: documentation/publish/start.md [hunk 7]
```diff
@@ -17,44 +17,41 @@ For your repository to be added there are a few criteria that need to be met.
...
+`zip_release` | bool | No | Indicates whether the content is in a zipped archive when releases are published on GitHub. If you use this you also need to add `filename`.
+`filename` | string | No | Name of the file HACS should look for, only applies to single item categories (plugin, theme, python_scripts, zip_release).
+`render_readme` | bool | No | Tells HACS to render the README.md file instead of info.md.
+`domains` | string | No | A list of domains.
+`country` | string | No | Two character country code in ISO 3166-1 alpha-2 format. [ISO 3166-1 alpha-2 on Wikipedia](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)
+`homeassistant` | string | No | The minimum required Home Assistant version.
+`persistent_directory` | string | No | A relative path (within the integration directory) that will be kept safe during upgrades. *Can only be used with integrations.*
+`iot_class` | string | Only for<br>integrations | The type of communication with the service/device. [Types used in Home Assistant](https://www.home-assistant.io/blog/2016/02/12/classifying-the-internet-of-things)
```
Comment (ludeeus) [inline documentation/publish/start.md:54]:
```suggestion
`iot_class` | string | Only for integrations | The type of communication with the service/device. [Types used in Home Assistant](https://www.home-assistant.io/blog/2016/02/12/classifying-the-internet-of-things)
```

This breaks the build

**Pull request merged**